src/main/java/by/exadel/internship/controller/UserController.java

```diff
@@ -65,6 +69,14 @@ public void restoreUser(@PathVariable UUID userId) {
     @ApiOperation("Save Users free time")
     public void saveUserTime(@PathVariable(name = "userId") UUID userId,
                              @RequestBody List<UserTimeSlotDTO> userTimeSlotDTOList) {
-        userTimeSlotService.saveUserTime(userTimeSlotDTOList,userId);
+        userTimeSlotService.saveUserTime(userTimeSlotDTOList, userId);
     }
+
+    @AdminAccessControl
+    @GetMapping("/skills")
+    @ApiOperation("Get user by skills")
+    public Set<UserDTO> getUserBySkills(@RequestParam("skills") List<Skill> skills) {
```

Comment:
What will happen if you pass invalid skill name or valid but with lower case mode for example?

Reply:
handled it. Check my code, please

Reply:
we will have 400 status, so we need to write right request , how to write our Skill we have GET endpoint for seeing it